src/js/components/App.vue

```diff
@@ -386,6 +683,73 @@ export default {
     }),
 
     methods: {
+        getMonth (birthdays) {
+            let date = new Date();
+
+            return birthdays.filter(birthday => {
+                return Number(birthday.date.split('-')[1]) === date.getMonth() + 1;
+            });
+        },
+
+        getToday (birthdays) {
+            let date = new Date();
+
+            return birthdays.filter(birthday => {
+                return Number(birthday.date.split('-')[0]) === date.getDate() && Number(birthday.date.split('-')[1]) === date.getMonth() + 1;
+            });
```

Comment:
The error that CI gave is this one: https://eslint.org/docs/rules/arrow-body-style

When you only have one simple command returning something inside the arrow function, it's preferred to direct return it:

```js
arg1 => {
    return arg1.toLowerCase();
}
```

Would turn into:
```js
arg1 => arg1.toLowerCase();
```

Take a look on this article: https://blog.mariusschulz.com/2015/06/09/returning-object-literals-from-arrow-functions-in-javascript

Reply:
I talked to @dbpolito regarding this getToday and getMonth methods. This is the case to use http://momentjs.com. We will use more things regarding to date on the future, so it's ok to import it